agent/app/agent.go

```diff
@@ -461,6 +439,12 @@ func (agent *ecsAgent) doStart(containerChangeEventStream *eventstream.EventStre
 		imageManager.AddImageToCleanUpExclusionList(agent.serviceconnectManager.GetLoadedImageName())
 	}
 
+	if csiDM, ok := agent.daemonManagers["ebs-csi-driver"]; ok {
```

Comment:
How about adding a constant for "ebs-csi-driver" as it is used in so many places?